components/ui/Toggle/Toggle.tsx

```diff
@@ -0,0 +1,43 @@
+import { cva, VariantProps } from "class-variance-authority";
+import { Switch } from "@headlessui/react";
+
+const toggleStyles = cva(
+  "relative inline-flex h-[38px] w-[74px] shrink-0 cursor-pointer rounded-full border-2 border-transparent transition-colors duration-200 ease-in-out focus:outline-none focus-visible:ring-2  focus-visible:ring-white focus-visible:ring-opacity-75",
+  {
+    variants: {
+      intent: {
+        lightEnabled: "bg-blue-500",
```

Comment:
let's use tailwind's `dark` selector 

Like [here](https://github.com/ninjha01/storybook/blob/b2275d3dc3accc68af584f70c5650cf0a2f35222/components/ui/Card/Card.tsx#L4)

and the builtin storybook control to switch the dark mode.

<img width="533" alt="image" src="https://user-images.githubusercontent.com/25281692/199791236-49796243-0cf7-4dc8-9de8-5d0fd7388fa3.png">

Reply:
I like the idea of having the `disabled` variant, so keep that!